float32 Pos_n
float32 Pos_e
float32 Pos_d
float32 V_n_ms
float32 V_e_ms
float32 V_d_ms
float32 u
float32 v
float32 w
float32 roll_deg
float32 pitch_deg
float32 yaw_deg
float32 p_body
float32 q_body
float32 r_body
float32 V_airspeed
float32 hour
float32 min
float32 sec
float32 Odometer
float32 CO2Density
float32 CO2alt_ft
float32 CO2lat_deg
float32 CO2lon_deg
float32 CO2heading_deg
float32 Temperature_degc
float32 Relative_humidity
float32 Pressure_inhg
float32 Dewpoint_degc
float32 wind_speed_kt
float32 wind_heading_deg
